package.json

```diff
@@ -4,15 +4,19 @@
   "description": "",
   "main": "js/server/server.js",
   "scripts": {
+    "start": "node server.js",
```

Comment:
Can you confirm that this works? Seems like it would need to be `js/server/server.js`

Reply:
not working